File: tests/C/0007.c
int main (int argc, char *argv[])
{
	int v1[100], v2[100], v3[100],v4[100];
	int v1a[100], v2a[100], v3a[100],v4a[100];

	#pragma acc update device(v1) host(v2)
	#pragma acc update device(v1) \
	 device(v2,v3) host(v4)

	#pragma acc update device(v1) host(v1a) \
	      device(v2) host(v2a) \
	      device(v3) host(v3a)

	return 0;
}

File: tests/Fortran/Free/0007.f90
program test
integer, parameter :: N = 10
real v1(N),v2(N),v3(N),v4(N)
real v1a(N),v2a(N),v3a(N)

!$acc update device(v1) host(v2)

!$acc update device(v1) &
!$acc device(v2,v3)&
!$acc host(v4)

!$acc update device(v1) host(v1a) &
!$acc& device(v2) host(v2a) &
!$acc& device(v3) host(v3a)

end program
